src/__tests__/patients/care-goals/CareGoalTab.test.tsx

```diff
@@ -18,30 +18,10 @@ import { RootState } from '../../../shared/store'
 const mockStore = createMockStore<RootState, any>([thunk])
 const { selectAll, arrowDown, enter } = specialChars
 
-type CareGoalTabWrapper = (store: any, history: MemoryHistory) => React.FC
-
-// eslint-disable-next-line react/prop-types
-const TabWrapper: CareGoalTabWrapper = (store, history) => ({ children }) => (
-  <Provider store={store}>
-    <Router history={history}>
-      <Route path="/patients/:id">{children}</Route>
-    </Router>
-  </Provider>
-)
-
-// eslint-disable-next-line react/prop-types
-const ViewWrapper: CareGoalTabWrapper = (store: any, history: MemoryHistory) => ({ children }) => (
-  <Provider store={store}>
-    <Router history={history}>
-      <Route path="/patients/:id/care-goals/:careGoalId">{children}</Route>
-    </Router>
-  </Provider>
-)
-
```

Comment:
Hah, nice! Thanks for cleaning up that code. I was thinking it was pretty nasty after I wrote it...